langworld_db_data/removers/listed_value_remover.py

```diff
@@ -70,23 +72,36 @@ def _remove_from_inventory_of_listed_values(
         print(f"Removed value {id_of_value_to_remove} from inventory of listed values")
         return value_to_remove
 
-    def _remove_from_feature_profiles(
+    def _remove_from_feature_profiles_and_update_ids_whose_indices_are_greater_than_one_of_removed_value(
         self,
         id_of_value_to_remove: str,
     ):
+        print(id_of_value_to_remove)
         for file in self.input_feature_profiles:
             is_changed = False
             rows = read_dicts_from_csv(file)
 
             for row in rows:
+                if row["value_type"] != "listed":
+                    continue
+                if row["feature_id"] != "-".join(id_of_value_to_remove.split("-")[:2]):
+                    continue
+
                 if row["value_id"] == id_of_value_to_remove:
                     row["value_id"] = ""
                     row["value_type"] = "custom"
+                    print(f"Changing value type to custom in {file.stem}")
+                    is_changed = True
+                    break
+                elif int(row["value_id"].split("-")[2]) > int(id_of_value_to_remove.split("-")[2]):
```

Comment:
```suggestion
                elif int(row["value_id"].split("-")[2]) > int(id_of_value_to_remove.split("-")[2]):
```
`id_of_value_to_remove.split("-")[2]` I think could be factored out as a constant

Reply:
Perhaps this can be improved within issue #310?

Reply:
OK, why not :)